.cirrus.yml

```diff
@@ -151,25 +150,15 @@ gating_task:
 # Ensure these container images can build
 container_image_build_task:
     alias: 'container_image_build'
+    name: "build gate image $DEST_BRANCH branch"
     depends_on:
         - "gating"
 
     # Only run for PRs, quay.io will automatically build after branch-push
     only_if: $CIRRUS_BRANCH != $DEST_BRANCH
 
-    matrix:
-        - name: "build in_podman image ${FEDORA_NAME} "
-          container:
-              dockerfile: Dockerfile
-        - name: "build in_podman image ${UBUNTU_NAME}"
-          container:
-              dockerfile: Dockerfile.ubuntu
-        - name: "build gate image $DEST_BRANCH branch"
-          container:
-              dockerfile: contrib/gate/Dockerfile
-
     container:
-        dockerfile: Dockerfile
+        dockerfile: contrib/gate/Dockerfile
```

Comment:
The string `Dockerfile.ubuntu` no longer appears anywhere in the source tree. Would it make sense to remove the top-level `Dockerfile.ubuntu` now? (It is much harder to check if it's also safe to remove the top-level `Dockerfile`, but I tried some heuristics and I think it might also be safe to remove it.

Reply:
Good catch, yes the Ubuntu dockerfile can go.  I'm afraid to remove the `Dockerfile` though, I seem to recall some build-related test (somewhere) needing at least a simple one.